src/components/Book/BookAbout/BookAbout.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import classes from './BookAbout.module.scss';
+import image from 'util/327752.jpeg';
+import { useParams } from 'react-router-dom';
+import Book from 'model/Book';
+import axios from 'axios';
+import Button from 'components/core/Button/Button';
+
+const backend_url = process.env.REACT_APP_BACKEND_URL;
+
+const BookAbout: React.FC = () => {
+  const { id } = useParams();
+  const [book, setBook] = useState<Book>({} as Book);
+
+  useEffect(() => {
+    axios.get(`${backend_url}/book/${id}`).then((res) => {
+      setBook(res.data);
+    });
+  }, [id]);
+
+  return (
+    <div className={classes['c-book-details']}>
+      <div className={classes['c-book-details__image-holder']}>
+        <img src={image} className={classes['c-book-details-image']} />
```

Comment:
Here it should be `c-book-details__image` as image is element of `c-book-details`